doc/cpp_domdec.rst

```diff
@@ -0,0 +1,186 @@
+Domain Decomposition
+====================
+
+Definitions
+-----------
+
+Domain decomposition
+    A description of the distribution of the model over the available
+    computational resources. The description partitions the
+    cells in the model as follows:
+
+        * group the cells into *cell groups* of the same kind of cell;
+        * assign each cell group to either a CPU core or GPU on a specific MPI rank.
+
+    The number of cells in each cell group depends on different factors,
+    including the type of the cell, and whether the cell group will run on a CPU
+    core or the GPU.
+
+    See :cpp:class:`arb::domain_decimposition`.
+
+Load balancer
+    A distributed algorithm that determines the domain decomposition using the
+    model recipe and and description of the available computational resources as
```

Comment:
and and

Reply:
Ah, yes, right you are. I thought that you were passively aggresively suggesting that more exposition was required ;)